packages/platform-core/src/telemetry/get_telemetry_url.ts

```diff
@@ -1,12 +1,11 @@
 import url, { UrlWithStringQuery } from 'node:url';
-import { latestApiVersion } from './constants.js';
 
 let cachedUrl: UrlWithStringQuery;
 
-const prodUrl = `https://api.cli.amplify.aws/${latestApiVersion}/metrics`;
+const prodUrl = `https://telemetry.cli.amplify.aws/metrics`;
```

Comment:
I was imagining sending telemetry to both endpoints for a while until we have good adoption for new versions carrying new telemetry. If we do a hard switch, we will have a tough time combining the metrics/data.